packages/core/src/components/DataTable/columns/renderDataColumns.tsx

```diff
@@ -71,12 +71,29 @@ export default function renderDataColumns<T>(
     const contents = React.createElement(customRenderer || DefaultRenderer, rendererArguments);
 
     return (
-      <div className={cx(styles && styles.row)}>
-        <div className={cx(styles && styles.row_inner)}>
-          <Spacing left={spacing} right={2}>
-            {contents || ''}
-          </Spacing>
-        </div>
+      <Spacing left={spacing} right={2}>
+        {contents || ''}
+      </Spacing>
+    );
+  };
+
+  const columnCellRenderer = (columnIdx: number) => (row: VirtualRow<T>) => {
+    const { dataKey, parent, rowIndex } = row;
+
+    const content = renderCell(dataKey, columnIdx, row);
+
+    return (
+      <div className={cx(styles.rowContainer)}>
+        <CellMeasurer
```

Comment:
if we are not using `dynamicRowHeight`, do you think we should still use this? or not really a problem?

Reply:
Good catch, doesn't seem like a problem, but probably better for performance.